### ISSUE TITLE
The searching not working with special Hungarian charachters

### ISSUE THREAD

**Issue by tibikepc** [AchillesPunks/repository.titan.bingie.mod#5]
Hello the searching options from tmdbhelper searching,not working with special hungarian characters.
For example i typing a movies name: gyonyoru eskuvo,the searching results is working and is good,but the correct name in hungarian language is gyönyörű esküvő, I type the correct name,gyönyörű esküvő and no searching results.
![20240519_221850](https://github.com/AchillesPunks/repository.titan.bingie.mod/assets/114329356/f8b95387-313c-40f5-ac44-560d969f7ff4)
![20240519_221650](https://github.com/AchillesPunks/repository.titan.bingie.mod/assets/114329356/0e436177-8a1f-4a5f-b97b-41e20d86e849)

**Comment by AchillesPunks** [comment]
You will need to translate the language file... I can not because I don't
know Hungarian. If you translate I'll add it to the skin.

**Comment by tibikepc** [comment]
Ok,I translate

**Comment by tibikepc** [comment]
Send me the language file

**Comment by AchillesPunks** [comment]
send me the file after you are done and I'll add it into the skin.. so if
there is another update it will already have your language

**Comment by tibikepc** [comment]
Ok,send me the language file.
tibikepc12@gmail.com

**Comment by AchillesPunks** [comment]
You already have the language file in Titan Bingie Mod.

[image: Screenshot (563).png]

**Comment by tibikepc** [comment]
This is the file ? 
![Screenshot_20240520_104540_Chrome](https://github.com/AchillesPunks/repository.titan.bingie.mod/assets/114329356/59f60d1d-c6a6-4911-a9f7-4ec9b7f52776)

**Comment by AchillesPunks** [comment]
Yes it's easy just edit from the install location on your system.
